README.md

```diff
@@ -114,9 +114,9 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
           <li><a href="docs/en_US/Assessor/BuiltinAssessor.md#Curvefitting">Curve Fitting</a></li>   
           </ul>
           </ul>
-          <a href="docs/en_US/Compressor/Overview.md">Model Compression (beta)</a>
+          <a href="docs/en_US/Compressor/Overview.md">Model Compression (Alpha)</a>
           <ul></ul>
-          <a href="https://github.com/microsoft/nni/blob/dev-nas-refactor/docs/en_US/NAS/Overview.md">NAS (beta)</a>
+          <a href="https://github.com/microsoft/nni/blob/dev-nas-refactor/docs/en_US/NAS/Overview.md">NAS (Alpha)</a>
```

Comment:
shall we merge the "Tuner for NAS" with this one?

Reply:
merged